modules/core/src/lib/model-utils.js

```diff
@@ -52,9 +60,6 @@ function mapAttributeName(name, options) {
 export function inferAttributeAccessor(attributeName, attribute) {
   let category;
   switch (attributeName) {
-    case 'indices':
```

Comment:
should we log a deprecated message/warning?

Reply:
The use case is still handled, I just moved the code.